src/colorprint.cc

```diff
@@ -136,13 +136,16 @@ void ColorPrintf(std::ostream& out, LogColor color, const char* fmt,
   CONSOLE_SCREEN_BUFFER_INFO buffer_info;
   GetConsoleScreenBufferInfo(stdout_handle, &buffer_info);
   const WORD old_color_attrs = buffer_info.wAttributes;
-
+  WORD  new_attr;
+  
   // We need to flush the stream buffers into the console before each
   // SetConsoleTextAttribute call lest it affect the text that is already
   // printed but has not yet reached the console.
   fflush(stdout);
+  new_attr = (buffer_info.wAttributes & ~7);
+  new_attr &= ~8;    // Since 'wAttributes' could have been hi-intensity at startup.
```

Comment:
@gvanem To clarify/reiterate, what happens if you literally apply the following change:
```suggestion
  new_attr = (buffer_info.wAttributes & ~FOREGROUND_INTENSITY);
  new_attr &= ~BACKGROUND_INTENSITY;    // Since 'wAttributes' could have been hi-intensity at startup.
```
Does it not compile?
The problem is that no one will have a clue what those `7` and `8` really meant,
and the docs imply that there are named macros, so we should use them.